Makefile

```diff
@@ -7,14 +7,15 @@ SHELL := /bin/bash
 
 TOP := $(shell pwd)
 
+DEPSSWIFT := ${TOP}/deps/swift
 DEPSBIN := ${TOP}/deps/bin
 DEPSGO := ${TOP}/deps/go
 DEPSTMP := ${TOP}/deps/tmp
-$(shell mkdir -p $(DEPSBIN) $(DEPSGO) $(DEPSTMP))
+$(shell mkdir -p $(DEPSBIN) $(DEPSGO) $(DEPSTMP) $(DEPSSWIFT))
 
-DEPSVENV := ${TOP}/deps/venv
+DEPSVENV := ${DEPSSWIFT}//deps/venv
 
-export PATH := ${DEPSBIN}:${DEPSGO}/bin:$(PATH)
+export PATH := ${DEPSBIN}/bin:${DEPSBIN}:${DEPSGO}/bin:$(PATH)
```

Comment:
DEPSBIN seems to have all binaries. Why do we need two folders?

Reply:
You are right.

Reply:
Mistake. Fixed this.